bootstrap.py

```diff
@@ -747,11 +738,31 @@ class MEOptions:
         print "\nExample usage: ./bootstrap.py -l admin -s foreman.example.com -o 'Default Organization' -L 'Default Location' -g My_Hostgroup -a My_Activation_Key"
         sys.exit(1)
 
+    # > Gather FQDN, HOSTNAME and DOMAIN using socket.getfqdn()
+    # > If socket.fqdn() returns an FQDN, derive HOSTNAME & DOMAIN using FQDN
+    # > else, HOSTNAME isn't an FQDN
+    # > if user passes --override-fqdn set FQDN, HOSTNAME and DOMAIN to the parameter that is given.
+    FQDN = socket.getfqdn()
```

Comment:
I think `--override-fqdn` should have `default=socket.getfqdn()` and the both codepaths should be merged to one?